test/resources/selinux.py

```diff
@@ -58,7 +58,7 @@
     "/var/lib/lxd(/.*)?",
     "/var/lib/microshift(/.*)?",
     "/var/lib/microshift-backups(/.*)?",
-    "/var/lib/microshift.saved(/.*)?",
+    r"/var/lib/microshift\.saved(/.*)?",
```

Comment:
There seems to be an unintentional leading "r"

Reply:
Good eye, that's actually intentional in this instance, because we have an escape character in the `\.saved` we want python to treat it as a raw string so it doesn't interpret the escape character, you do that by prepending the `r` in front of a string to denote that.